quickstart/Makefile

```diff
@@ -7,6 +7,13 @@
 # Determine the operating system
 KERNEL_NAME := $(shell uname -s)
 
+# Set appropriate date command based on OS
+ifeq ($(KERNEL_NAME),Darwin)
+    DATE_CMD = gdate
+else
+    DATE_CMD = date
+endif
+
```

Comment:
If we are going to use nix to provide `date` via the `coreutils` package, then we shouldn't need to flip between `gdate` and `date` based on the OS.  Further more I don't have `gdate` available on my system.